regulations/generator/html_builder.py

```diff
@@ -229,3 +234,19 @@ def process_node(self, node, indexes=None):
         label_id = '-'.join(node['label'])
         node['accepts_comments'] = label_id in self.diff_applier.diff
         node['comments_calledout'] = label_id in self.diff_applier.diff
+
+        node['has_diff'] = label_id in self.diff_applier.diff
```

Comment:
AFAICT, these fields have a one-to-one mapping with `data-stars` attributes in the templates and don't get used otherwise. I'm wondering if it would be easier for a maintainer to understand the flow from `HTMLBuilder` to the templates to the js if we used more consistent names.